### PR TITLE
fix(windrose): Fix mesh size bug, and frequency_maximum to interval.

### PR DESCRIPTION
**Pull request by saeranv**
@chriswmackey 

This PR contains the bug fix for the windmesh scaling issue I mentioned in my previous PR. The windrose is now scaling correctly with and without the calmrose. 

Also I changed the way users control the maximum frequency from them setting maximum hours, to setting number of frequency lines in the windrose plot, which should be a lot more intuitive.

### CODE REVIEW

File: ladybug/windrose.py [hunk 1]
```diff
@@ -222,8 +213,25 @@ def frequency_hours(self, frequency_hours):
         self._frequency_hours = int(frequency_hours)
 
     @property
-    def frequency_intervals(self):
-        return int(math.ceil(self.frequency_maximum / self.frequency_hours))
+    def frequency_intervals_compass(self):
+        """Calculate the number of intervals in the windrose compass."""
```
Comment (chriswmackey) [inline ladybug/windrose.py:217]: This doesn't seem like the right type of docstring for a property. Property docstrings should start with "Get" or "Get or set" depending on whether they are set-able.

File: ladybug/windrose.py [hunk 2]
```diff
@@ -222,8 +213,25 @@ def frequency_hours(self, frequency_hours):
...
+        if self._frequency_intervals_compass is None:
+            self._frequency_intervals_compass = self.frequency_intervals_mesh
+        return self._frequency_intervals_compass
+
+    @frequency_intervals_compass.setter
+    def frequency_intervals_compass(self, frequency_intervals_compass):
+        """Calculate the number of intervals in the windrose compass."""
```
Comment (chriswmackey) [inline ladybug/windrose.py:224]: Same comment here.

File: ladybug/windrose.py [hunk 3]
```diff
@@ -222,8 +213,25 @@ def frequency_hours(self, frequency_hours):
...
+        assert frequency_intervals_compass >= 1, 'The frequency_intervals_compass ' \
+            'must be greater then 0. Got: {}'.format(frequency_intervals_compass)
+        self._compass = None
+        self._container = None
+        self._frequency_intervals_compass = int(frequency_intervals_compass)
+
+    @property
+    def frequency_intervals_mesh(self):
+        """Calculate the number of intervals in the windrose mesh."""
```
Comment (chriswmackey) [inline ladybug/windrose.py:233]: And here.

**Comment by saeranv** [comment]
@chriswmackey 

Thanks, I realized that I really haven't been following that rule consistently so went through and fixed it in all my properties.

**Comment by chriswmackey** [comment]
Great! Let's merge.

**Pull request merged**